airflow/models/serialized_dag.py

```diff
@@ -37,10 +37,25 @@
 from airflow.utils import timezone
 from airflow.utils.session import provide_session
 from airflow.utils.sqlalchemy import UtcDateTime
+from airflow.www.security import AirflowSecurityManager
 
 log = logging.getLogger(__name__)
 
 
+class SimpleSecurityManager(AirflowSecurityManager):
```

Comment:
@jhtimmins, curious is you know a better way or trick to using the security manager somewhere where we don't want/need the whole flask app?

Reply:
@jedcunningham Oof I need to think about this, because generally speaking we really don't want to extend the webserver-level controls into Airflow core.

The ability to define access controls inside the DAG is a huge antipattern imo.